generated/nirfmxwcdma/nirfmxwcdma.proto

```diff
@@ -13,53 +13,3435 @@ option csharp_namespace = "NationalInstruments.Grpc.NiRFmxWCDMA";
+}
+
+enum ModAccPeakCdeBranch {
+  MODACC_PEAK_CDE_BRANCH_I = 0;
+  MODACC_PEAK_CDE_BRANCH_Q = 1;
+  MODACC_PEAK_CDE_BRANCH_I_AND_Q = 2;
+}
+
+enum ModAccPeakRcdeBranch {
+  MODACC_PEAK_RCDE_BRANCH_I = 0;
+  MODACC_PEAK_RCDE_BRANCH_Q = 1;
+  MODACC_PEAK_RCDE_BRANCH_I_AND_Q = 2;
+}
+
+enum ModAccSynchronizationMode {
+  MODACC_SYNCHRONIZATION_MODE_FRAME = 0;
+  MODACC_SYNCHRONIZATION_MODE_SLOT = 1;
+  MODACC_SYNCHRONIZATION_MODE_ARBITRARY = 2;
+  MODACC_SYNCHRONIZATION_MODE_MARKER = 3;
+}
+
+enum ModulationType {
+  MODULATION_TYPE_BPSK_QPSK = 0;
+  MODULATION_TYPE_4PAM_16QAM = 1;
+  MODULATION_TYPE_64QAM = 2;
+}
+
+enum ObwAveragingEnabled {
+  OBW_AVERAGING_ENABLED_FALSE = 0;
+  OBW_AVERAGING_ENABLED_TRUE = 1;
+}
+
+enum ObwAveragingType {
+  OBW_AVERAGING_TYPE_RMS = 0;
+  OBW_AVERAGING_TYPE_LOG = 1;
+  OBW_AVERAGING_TYPE_SCALAR = 2;
+  OBW_AVERAGING_TYPE_MAX = 3;
+  OBW_AVERAGING_TYPE_MIN = 4;
+}
+
+enum ObwRbwAutoBandwidth {
+  OBW_RBW_AUTO_BANDWIDTH_FALSE = 0;
+  OBW_RBW_AUTO_BANDWIDTH_TRUE = 1;
+}
+
+enum ObwRbwFilterType {
+  OBW_RBW_FILTER_TYPE_FFT_BASED = 0;
+  OBW_RBW_FILTER_TYPE_GAUSSIAN = 1;
+  OBW_RBW_FILTER_TYPE_FLAT = 2;
+}
+
+enum ObwSweepTimeAuto {
+  OBW_SWEEP_TIME_AUTO_FALSE = 0;
+  OBW_SWEEP_TIME_AUTO_TRUE = 1;
+}
+
+enum QevmAveragingEnabled {
+  QEVM_AVERAGING_ENABLED_FALSE = 0;
+  QEVM_AVERAGING_ENABLED_TRUE = 1;
+}
+
+enum RFAttenuationAuto {
+  RF_ATTENUATION_AUTO_FALSE = 0;
+  RF_ATTENUATION_AUTO_TRUE = 1;
+}
+
+enum SemAveragingEnabled {
+  SEM_AVERAGING_ENABLED_FALSE = 0;
+  SEM_AVERAGING_ENABLED_TRUE = 1;
+}
+
+enum SemAveragingType {
+  SEM_AVERAGING_TYPE_RMS = 0;
+  SEM_AVERAGING_TYPE_LOG = 1;
+  SEM_AVERAGING_TYPE_SCALAR = 2;
+  SEM_AVERAGING_TYPE_MAX = 3;
+  SEM_AVERAGING_TYPE_MIN = 4;
+}
+
+enum SemLowerOffsetMeasurementStatus {
+  SEM_LOWER_OFFSET_MEASUREMENT_STATUS_FAIL = 0;
+  SEM_LOWER_OFFSET_MEASUREMENT_STATUS_PASS = 1;
+}
+
+enum SemMeasurementStatus {
+  SEM_MEASUREMENT_STATUS_FAIL = 0;
+  SEM_MEASUREMENT_STATUS_PASS = 1;
+}
+
+enum SemSweepTimeAuto {
+  SEM_SWEEP_TIME_AUTO_FALSE = 0;
+  SEM_SWEEP_TIME_AUTO_TRUE = 1;
+}
+
+enum SemUpperOffsetMeasurementStatus {
+  SEM_UPPER_OFFSET_MEASUREMENT_STATUS_FAIL = 0;
+  SEM_UPPER_OFFSET_MEASUREMENT_STATUS_PASS = 1;
+}
+
+enum SlotPhaseSynchronizationMode {
+  SLOT_PHASE_SYNCHRONIZATION_MODE_FRAME = 0;
+  SLOT_PHASE_SYNCHRONIZATION_MODE_SLOT = 1;
+}
+
+enum SlotPowerSynchronizationMode {
+  SLOT_POWER_SYNCHRONIZATION_MODE_FRAME = 0;
+  SLOT_POWER_SYNCHRONIZATION_MODE_SLOT = 1;
+}
+
+enum TriggerMinimumQuietTimeMode {
+  TRIGGER_MINIMUM_QUIET_TIME_MODE_MANUAL = 0;
+  TRIGGER_MINIMUM_QUIET_TIME_MODE_AUTO = 1;
+}
+
+enum UplinkScramblingType {
+  UPLINK_SCRAMBLING_TYPE_LONG = 0;
+  UPLINK_SCRAMBLING_TYPE_SHORT = 1;
+}
+
+enum UplinkTestModel {
+  UPLINK_TEST_MODEL_R6C_2_1 = 0;
+  UPLINK_TEST_MODEL_R6C_2_2 = 1;
+  UPLINK_TEST_MODEL_R6C_2_3 = 2;
+  UPLINK_TEST_MODEL_R6C_2_4 = 3;
+  UPLINK_TEST_MODEL_R6C_2_5 = 4;
+  UPLINK_TEST_MODEL_R6C_10_1_4_SUBTEST1 = 5;
```

Comment:
As far as I know, these are correct and fine. But this kind of numeric/acronym blast often gets messed up by codegen.

Reply:
Yes, this kind of numeric/acronym is strange.  Do we need some actions to handle such problem?